README.md

```diff
@@ -763,6 +763,7 @@ metadata in media files, including video, audio, and photo formats
 * [ExpressMapper](https://github.com/fluentsprings/ExpressMapper) - A lightweight, lighting fast .NET mapper to map one type of object(s) to another in automated and easy way. ExpressMapper relies completely on the expression trees.
 * [AgileMapper](https://github.com/agileobjects/AgileMapper) - A zero-configuration Object-Object mapper supporting .NET Standard 1.0
 * [Mapster](https://github.com/MapsterMapper/Mapster) - A high performance object mapper in .net 
+
```

Comment:
:(

Reply:
The rest of these subheadings are separated by new line.
I noticed that the "Office" subheading is not so I've added a quick fix.

Reply:
:)